newdle/api.py

```diff
@@ -291,6 +292,21 @@ def get_newdles_participating():
     return NewdleParticipantSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdle/event', methods=('POST',))
+@use_kwargs(NewEventSchema(), locations=('json',))
+def create_event(title, duration, final_dt):
+    url = current_app.config['INDICO_URL']
+    res = requests.post(
+        url,
+        data={
+            'title': title,
+            'start_dt': final_dt,
+            'duration': duration,
+        },
+    )
+    return res.text
```

Comment:
You are returning `text/html` here atm. I'd rather `return jsonify(url=res.text)` here. Or even better, make the Indico API also return JSON and then use `res.json()['url']`

Reply:
Thanks, I'm returning `res.json()` now and accessing the URL with `resp.url` as before.